CRM/Contact/Page/View/Relationship.php

```diff
@@ -134,6 +134,9 @@ public function view() {
    */
   public function browse() {
     // do nothing :) we are using datatable for rendering relationship selectors
+    $columnHeaders = CRM_Contact_BAO_Relationship::getColumnHeaders();
+    CRM_Utils_Hook::searchColumns('relationship.columns', $columnHeaders, $contactRelationships, $null);
```

Comment:
$contactRelationships and $null is undefined, we might get notice errors.

Reply:
Thanks @pradpnayak I've changed that